core/arch/arm/plat-ls/ls_hw_unq_key.S

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Missing SPDX license identifier and please just add "Copyright 2018 NXP" (no other legal blurb).
See https://github.com/OP-TEE/optee_os/blob/master/documentation/copyright_and_license_headers.rst.

Reply:
Ok.